src/cook-web/src/app/c/[[...id]]/Components/ChatUi/ListenModeRenderer.scss

```
@@ -394,6 +394,162 @@
+    padding: 36px 84px 0 36px;
+  }
+
+  .slide-player__interaction-title {
+    font-size: 22px;
+    line-height: 30px;
+  }
+
+  .slide-player__interaction-body {
+    --mdf-slide-font-size: 24px;
+    --mdf-slide-line-height: 36px;
+    box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)